appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -953,14 +953,18 @@
   @Description("Checkbox controlling whether to display invisible components in the designer.")
   String showHiddenComponentsCheckbox();
 
-  @DefaultMessage("Check to see Preview on Tablet size.")
-  @Description("Checkbox (check) controlling whether to display a preview on Tablet size.")
+  @DefaultMessage("Select to see Preview on Tablet size.(320,470)")
```

Comment:
I think you have the numbers for tablet and phone backwards. The numbers here are smaller than the ones given for phone screen below. I would also recommend that if we are giving the dimensions, we don't include the period here before the opening parenthesis.

Reply:
For the phone pixel size and tablet pixel size, I got them from MockForm.java, which were predefined by other programmers I think. I will change the monitor size to more reasonable values.
Besides, what do you mean by not include the period?